spot-client/src/common/ui/components/code-input/code-input.js

```diff
@@ -3,22 +3,19 @@ import React from 'react';
 
 import { isAutoFocusSupported } from 'common/utils';
 
-import InputBox from './input-box';
-
 /**
  * A mapping of keyboard key names to keyboard key codes for readability while
  * processing key codes.
  */
 const keyCodes = {
-    BACKSPACE_KEY: 8,
     LEFT_ARROW_KEY: 37,
```

Comment:
Do you still need these?

Reply:
I use them to prevent manual cursor movement on browsers, so the cursor is always at the end of the inputed value.